src/containers/checkout/delivery-type/delivery-type.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from 'react';
+
+import { FormControl, FormControlLabel, Radio, RadioGroup } from '@material-ui/core';
+
+import { useDispatch } from 'react-redux';
+import { CHECKOUT_TITLES, DELIVERY_TYPE } from '../../../translations/checkout.translations';
+
+import { useStyles } from './delivery-type.styles';
+import { addDeliveryType } from '../../../redux/cart/cart.actions';
+import { deliveryTypes, SESSION_STORAGE } from '../../../configs';
+import { getFromSessionStorage } from '../../../services/session-storage.service';
+import { setDeliveryTypeToStorage } from '../../../utils/checkout';
+
+const DeliveryType = ({ language }) => {
```

Comment:
This is new component so I suggest to use i18n for translation, cause we are refactoring it now.